server/src/main/java/org/opensearch/common/settings/KeyStoreWrapper.java

```diff
@@ -523,6 +523,7 @@ private void decryptLegacyEntries() throws GeneralSecurityException, IOException
     /** Write the keystore to the given config directory. */
     public synchronized void save(Path configDir, char[] password) throws Exception {
         ensureOpen();
+        assert password.length >= 20;
```

Comment:
if internal keystore is not set, application creates its own without password, but BCFIPS won't allow empty password for keystore.